src/trackCopiedText.js

```diff
@@ -1,18 +1,57 @@
-/**
- * Tracking text copied by the user.
- * 
- * Adds event listener that monitors all copy events and sends text copied at the time as custom event.
- */
-
- export default (subscribe) => {
-    
-    const processCopyEvent = () => {
-        let copiedItemText = getSelectedText();
-        var eventData = ["trackEvent","User interaction","Copying text",copiedItemText];
-        subscribe(eventData);   
-    };
-    
-    const getSelectedText = () => window.getSelection()?.toString() ?? '';
-    
-    window.addEventListener("copy", processCopyEvent);
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck

```

Comment:
Because TS complains about that webpack will do so also. Unfortunetly this flag doesnt help for some reason. Currently this script is not built at all, I would like to suggest to disable TS checking on webpack config at all as we do not have TS in that project but only uses ts-loader. To do that we have to add proper lines in [webpack config file](https://github.com/PiwikPRO/solutions-templates/blob/PPREVOPS-3158-remove-excessive-scroll/webpack.config.js#L48) :
```
{
            test: /\.(ts|tsx|jsx|js)$/,
            loader: require.resolve('ts-loader'),
            options: {
              transpileOnly: true,
            }
          },
```

Reply:
added in the latest commit